language/stdlib/modules/libra_account.mvir

```diff
@@ -278,13 +278,37 @@ module LibraAccount {
     // Creates a new account at `fresh_address` with the `initial_balance` deducted from the
     // transaction sender's account
     public create_new_account(fresh_address: address, initial_balance: u64) acquires T {
-        create_account(copy(fresh_address));
+        let auth_key: bytearray;
+        let zero_balance: LibraCoin.T;
+        let sent_handle: Event.Handle<Self.SentPaymentEvent>;
+        let received_handle: Event.Handle<Self.ReceivedPaymentEvent>;
+        let account: Self.T;
+
+        auth_key = AddressUtil.address_to_bytes(copy(fresh_address));
+        sent_handle = Event.new_event_handle<Self.SentPaymentEvent>();
+        received_handle = Event.new_event_handle<Self.ReceivedPaymentEvent>();
+        zero_balance = LibraCoin.zero();
+
+        account = T {
+            authentication_key: move(auth_key),
+            balance: move(zero_balance),
+            delegated_key_rotation_capability: false,
+            delegated_withdrawal_capability: false,
+            received_events: move(received_handle),
+            sent_events: move(sent_handle),
```

Comment:
You don't need these locals if you don't want them (i.e. `authentication_key: AddressUtil.address_to_bytes(copy(fresh_address))` works if you want to use it)